odoo/addons/base/models/res_users.py

```diff
@@ -523,7 +523,7 @@ def read(self, fields=None, load='_classic_read'):
         if fields and self == self.env.user:
             readable = self.SELF_READABLE_FIELDS
             for key in fields:
-                if not (key in readable or key.startswith('context_')):
+                if not (key in readable):
```

Comment:
huh?

Reply:
It is a mistake, we wanted to remove the one in `project`. 🙂 
I revert this diff. ^^